tests/contrib/django/test_restframework.py

```diff
@@ -0,0 +1,53 @@
+import django
+from django.apps import apps
+from nose.tools import ok_, eq_
+from unittest import skipIf
+
+from .utils import DjangoTraceTestCase
+
+@skipIf(django.VERSION < (1, 10), 'requires django version >= 1.10')
+class RestFrameworkTest(DjangoTraceTestCase):
+    def setUp(self):
+        super(RestFrameworkTest, self).setUp()
+
+        # We do the imports here because importing rest_framework with an older version of Django 
+        # would raise an exception
+        from rest_framework.views import APIView
+        from ddtrace.contrib.django.restframework import ORIGINAL_HANDLE_EXCEPTION, unpatch_rest_framework
+
+        self.APIView = APIView
+        self.ORIGINAL_HANDLE_EXCEPTION = ORIGINAL_HANDLE_EXCEPTION
+        self.unpatch_rest_framework = unpatch_rest_framework
+
+    def test_setup(self):
+        ok_(apps.is_installed('rest_framework'))
+        ok_(hasattr(self.APIView, self.ORIGINAL_HANDLE_EXCEPTION))
+
+    def test_unpatch(self):
```

Comment:
Can you check that no span is generated if you raise an exception after it is unpatched?

Reply:
A span will be generated, but the error won't get caught.

Reply:
Right sorry. Can you check that behavior then? It would validate that your instrumentation changes the existing behavior.